### PR TITLE
Added aria-hidden="true"

### PR DESCRIPTION
**Pull request by N8Solutions**
Added aria-hidden="true" where font awesome icons are referenced in the files. These instances were all I could find. This does not mean there are not more that I may have missed.

### CODE REVIEW

**Comment by mahagr** [comment]
I need some time to review all the changes :) Thanks.

**Comment by N8Solutions** [comment]
Of course, that's to be expected! It took me several hours over the course of 2 days to find and add them so I wouldn't expect you to review and approve them instantly or even quickly for that matter.

File: platforms/common/application/changelog/index.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ ready(function() {
                         section.parent('li').after(section.parent('ol').find('> li'));
                     }
 
-                    zen('i[class="fa g-changelog-toggle fa-fw fa-' + status + '"]').bottom(section);
+                    zen('i[class="fa g-changelog-toggle fa-fw fa-' + status + '"][aria-hidden="true"]').bottom(section);
```
Comment (mahagr) [inline platforms/common/application/changelog/index.js]: @w00fz Is this correct?

Comment (w00fz) [inline platforms/common/application/changelog/index.js]: Yes all good

**Comment by N8Solutions** [comment]
Thank you! Glad to help contribute to the project any way I can.

**Pull request merged**